cluster/libvirt-coreos/util.sh

```diff
@@ -51,6 +52,19 @@ function detect-nodes {
   KUBE_NODE_IP_ADDRESSES=("${NODE_IPS[@]}")
 }
 
+function set_service_accounts {
+    SERVICE_ACCOUNT_KEY=${SERVICE_ACCOUNT_KEY:-"/tmp/kube-serviceaccount.key"}
+    # Generate ServiceAccount key if needed
+    if [[ ! -f "${SERVICE_ACCOUNT_KEY}" ]]; then
+      mkdir -p "$(dirname ${SERVICE_ACCOUNT_KEY})"
+      openssl genrsa -out "${SERVICE_ACCOUNT_KEY}" 2048 2>/dev/null
```

Comment:
`libvirt-coreos` targets local development environments.
I think it is reasonable to request `openssl` as a pre-requisite. It should be easy for the developer to install it on its workstation.
But we could document this new dependency in: https://github.com/kubernetes/kubernetes/blob/master/docs/getting-started-guides/libvirt-coreos.md#prerequisites